static/styles/compose.css

```diff
@@ -109,11 +109,28 @@
             position: relative;
             margin: 0 5px;
         }
+
+        button.narrow_to_compose_recipients {
+            background: transparent;
+            font-size: 20px;
+            line-height: 20px;
+            opacity: 0.6;
+            border: 0;
+
+            &:hover {
+                opacity: 1;
+            }
+
+            &:focus,
```

Comment:
We want focus outline here for the icon.

Reply:
Can we just have outlines with the keyboard but not on click?

Reply:
The icon looks too glued to the topic input box in this iteration. We need more left margin. The 1px padding around the icon for the outline looks good.

Reply:
Does this look good? 
![image](https://user-images.githubusercontent.com/44665669/158538571-22cbee98-881b-4380-b390-aa7ee63de92a.png)
added `margin: 3px`.
![image](https://user-images.githubusercontent.com/44665669/158538674-790bcc1e-d0c2-468b-9fdf-c5f23508db01.png)

Reply:
@alya Yes, changed to that behavior. Thanks @amanagr, I didn't know changing to `a` tag fixes it. How does it work? Shouldn't we be able to achieve this behavior on anything with styling on `:focus-visible`? (It didn't work for button for me)